core/services/ping/pingmanager.py

```diff
@@ -27,15 +27,27 @@ async def register_ethernet_ping360(self, ping: PingDeviceDescriptor) -> None:
         if ping not in self.drivers:
             self.drivers[ping] = Ping360Driver(ping, -1)
 
+    def find_next_port(self, base_port: int, direction: int) -> int:
```

Comment:
Why not using port `0` to bind and get the port that the OS found for us ?

Reply:
I prefer following some kind of order for now...

Reply:
also, we're starting at the ports checked by pingviewer for now. we still dont´t have the discovery protocol implemented for alternative ports